homeassistant/components/decora_wifi/fan.py

```diff
@@ -0,0 +1,128 @@
+"""Support for Decora Wifi Fan controls."""
```

Comment:
Ah PR should be as minimal as possible and not introduce multiple new things at once.
Please reduce this PR to a single thing and do other new features in follow-ups.

Reply:
Ok; the fan stuff is in a separate commit so that should be pretty straightforward.  Sorry for the git noob question, but what is the best way to go about doing that so as not to muck up the git history?  I made a new branch in my repository called decora_wifi-config-flow-only without the fan support commit; should I close this PR and submit a new one from that branch? Or is there a better option I'm not seeing?

Reply:
In this case, it will be conflicting, so the best is to keep the PR limited to the configuration flow and open up a PR for the fan stuff after this one has been merged.

Reply:
Ok, fan stuff removed.